pkg/block/metadata/meta.go

```diff
@@ -59,6 +59,9 @@ type Thanos struct {
 
 	// Source is a real upload source of the block.
 	Source SourceType `json:"source"`
+
+	// Hashes is a map of hashes inside of the block.
```

Comment:
I guess you mean, `// Hashes is a map of hashes of files inside the block.`

Reply:
:smile: yes, indeed. Thank you for spotting this! :heart: